---
title: Research Interests
layout: page
---
# Research Interests

My research focuses on four closely related areas in computational biology and bioinformatics: 1) Development of innovative computational methods for knowledge discovery. I pioneer the development of novel computational tools and artificial intelligence algorithms to incorporate highly heterogeneous big data for novel knowledge discovery; 2) Multi-omics integration and interpretation of molecular intricacy to derive actionable insights; 3) Application of data-driven approaches for deciphering disease mechanisms; and 4) Integrative approach for patient characterization and precision medicine.

## Development of innovative computational methods for knowledge discovery. 

Our developed methods contribute to the interpretation of heterogeneous biological data for knowledge discovery. I have developed tailored methods dealing with start-of-art biological data, including: 1) the accurate alignment of single-cells from different platforms, species, and omics layers; 2) transferring single-cell information to disentangle the cell compositions in spot-level spatial resolved transcriptomics; and 3) the statistical modeling of single-cell data from different sources for novel comparative analysis. These studies provide fundamental work for developing novel algorithms to further address missing data issues in single-cell epigenomics and spatial transcriptomics field.
   

##	Multi-omics integration and interpretation of molecular intricacy to derive actionable insights. 

Our developed methods contribute to the understanding of the interplay of different molecule layers in diseases. These work include: 1) the joint statistical method for integrative analysis of single-cell multi-omics data, which enables to detect coherent regulatory signals and target genes from unpaired scRNA-seq and scATAC-seq data; 2) the integrative approach of bioinformatics and multi-scale modeling on scRNA-seq and proteomics data, which predicts the dynamic drug response of lung tumor cells; and 3) the sNMF-based subtyping approach that deciphers the biomarkers of brain metastasis based on transcriptomic, proteomic and metabolomic data.

   
## Application of data-driven approaches for deciphering disease mechanisms. 

My work of using data-driven approaches have contributed to the understanding of underlying mechanisms in different diseases. 1) To reveal the underlying activities in lung tumor microenvironment, I computationally analyzed the single-cell data from lung cancer patients and revealed intercellular myeloid cells reprogramming whereby tumor cells stimulated monocyte‐to‐M2 differentiation. 2) I also investigated the potential functions of fibroblasts in hepatocellular carcinoma. 3) To enhance checkpoint immunotherapy in melanoma patients, I applied a computational predictive approach and identified B-cell biomarker IGLC3 associated with the progression of individual melanoma patient in response to checkpoint therapy.


## Integrative approach for patient characterization and precision medicine. 

1) I have made impactful contributions to the patient characterization and subtyping based on multi-modal data types, including clinical, copy number alterations, mutations, DNA methylation, and mRNA expression, which identified the clinically significant subtypes and suggested potential therapeutic targets; 2) During the COVID-19 pandemic, I have been working on characterizing the cancer patients within NCATS’ National COVID Cohort Collaborative, to investigate the risk factors of COVID-19 breakthrough infections and severe outcomes. My study highlighted patients who received recent immunotherapies or targeted therapies did not have higher risk of overall mortality or breakthrough infection risks, and provided evidence on the effects of COVID-19 on cancer outcomes and the ability to continue specific cancer treatments. This work revealed fundamental insights valuable to the precision protection of the cancer population.
